packages/terra-data-grid/src/proptypes/rowShape.js

```diff
@@ -8,19 +8,17 @@ const rowShape = PropTypes.shape({
   id: PropTypes.string.isRequired,
 
   /**
-   * Data to be displayed in the cells of the row. Cells will be rendered in the row in the order given.
+   * An array of cell objects that define the content to be rendered in the row. Cells will be rendered in the order given and are expected to be in the same order as the columns.
    */
   cells: PropTypes.arrayOf(cellShape),
   /**
    * A boolean indicating whether or not the row should render as selected.
    */
   isSelected: PropTypes.bool,
   /**
-   * A string identifier used to describe the row contents. This value will be used to construct additional labels
-   * for internal controls (e.g. row selection cells).
+   * A string identifier used to describe the row contents. This value will be used by screen readers when announcing the row (un)selection.
```

Comment:
Do we usually refer to this as "unselection" or "deselection"? If it's the latter, can we update to:

```suggestion
   * A string identifier used to describe the row contents. This value will be used by screen readers when announcing the row deselection.
```

Reply:
We prefer to keep the current language.

Reply:
Ok I think I misunderstood the original meaning of this to be that the value is only used for deselection. If it's being used by screen readers for both selection and unselection, can we please reword to:

```suggestion
   * A string identifier used to describe the row contents. This value will be used by screen readers when announcing row selection and unselection.
```